app/presenters/taxon_presenter.rb

```diff
@@ -37,4 +37,8 @@ def options_for_child_taxon(index:)
       track_options: {},
     }
   end
+
+  def is_brexit?
```

Comment:
Minor: You don't need the `is_` prefix in the method name, that's covered by using `?`

Reply:
Thanks, Leena!